bpf/ut/bpf_prog_test.go

```diff
@@ -309,12 +311,13 @@ func initMapsOnce() {
 		ipsMap = ipsets.Map(mc)
 		stateMap = state.Map(mc)
 		testStateMap = state.MapForTest(mc)
-		jumpMap = jump.MapForTest(mc)
+		xdpJumpMap = MapForTest(mc)
```

Comment:
nit: maybe move into its own package, e.g. "xdpjump" ?

Reply:
The only thing is that the xdpjump is only needed for ut. We don't need anywhere else. For that, I felt maybe it is better to define it somewhere else. Does it make sense?

Reply:
OK, no worries, that's absolutely fine.

Reply:
Well, just realised that the extra map is only needed for "runBpfUnitTest", so it is also removed now.